web_developer_mode_optional/static/description/icon.svg

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
You already have the source in https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg, that's why I ask not to duplicate the file unless you make any modification.